lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -8,7 +8,9 @@ def compound_changelog
       sort_tags_by_date(@filtered_tags)
       fetch_issues_and_pr
 
-      log = "#{@options[:header]}\n\n"
+      log = ''
+      log += @options[:frontmatter] if @options[:frontmatter]
```

Comment:
Sure. I was wondering about lumping them all together, but testing without the caching became incredibly frustrating.